cob_cartesian_controller/CMakeLists.txt

```diff
@@ -85,6 +80,10 @@ install(DIRECTORY include/${PROJECT_NAME}/
  PATTERN ".svn" EXCLUDE
 )
 
-install(DIRECTORY launch movement
+install(PROGRAMS scripts/test_move_lin.py scripts/test_move_circ.py scripts/test_move_santa.py
```

Comment:
Is it really necessary to install the "test_*" scripts? Should be enough to provide them in developer space without installation.